lib/go/test/service_test.go

```diff
@@ -45,38 +45,24 @@ func TestContracts(t *testing.T) {
 
 	env.StorageFeesAddress = storageFeesAddress.String()
 
-	result, err := b.ExecuteScript(templates.GenerateGetStorageFeeConversionScript(env), nil)
-	require.NoError(t, err)
-	if !assert.True(t, result.Succeeded()) {
-		t.Log(result.Error.Error())
-	}
-	conversion := result.Value
-	assertEqual(t, CadenceUFix64("1.0"), conversion.(cadence.UFix64))
+	t.Run("Should have initialized fields correctly", func(t *testing.T) {
 
-	result, err = b.ExecuteScript(templates.GenerateGetStorageFeeMinimumScript(env), nil)
-	require.NoError(t, err)
-	if !assert.True(t, result.Succeeded()) {
-		t.Log(result.Error.Error())
-	}
-	min := result.Value
-	assertEqual(t, CadenceUFix64("0.0"), min.(cadence.UFix64))
+		result := executeScriptAndCheck(t, b, templates.GenerateGetStorageFeeConversionScript(env), nil)
+		assertEqual(t, CadenceUFix64("1.0"), result.(cadence.UFix64))
```

Comment:
Try to remove the casts in equality assertions, i.e. just write
```suggestion
		assertEqual(t, CadenceUFix64("1.0"), result)
```

If the result is not a `cadence.UFix64`, the current assertion will crash the test.
The assertion already checks the types match.